providers/os/resources/lsblk.go

```diff
@@ -34,7 +35,19 @@ func (l *mqlLsblk) list() ([]interface{}, error) {
 		d := blockEntries.Blockdevices[i]
 		for i := range d.Children {
 			entry := d.Children[i]
-			entry.Mountpoints = append(entry.Mountpoints, entry.Mountpoint)
+			// Some versions of the lsblk return [null] instead of empty array
+			entry.Mountpoints = slices.Collect(func(yield func(interface{}) bool) {
```

Comment:
Should this happen inside the parsing of the mount block entries?

Reply:
Good point, will update